DOCSP-48562: dupe index config (#117)
mongodb/docs-mongoid: source/whats-new.txt

Learn what's new in:

* :ref:`Version 9.1 <mongoid-version-9.1>`
* :ref:`Version 9.0 <mongoid-version-9.0>`

To view a list of releases and detailed release notes, see {+odm+}
:github:`Releases <mongodb/mongoid/releases>` on GitHub.

.. _mongoid-version-9.1:

What's New in 9.1
-----------------

The 9.1 release includes the following new features, improvements, and
fixes:

- Creates the ``Mongoid.allow_duplicate_index_declarations``
  configuration option that enables you to send index verification to

>   the server. This allows you to view an error if a duplicate index is
>   declared. To learn more, see the :ref:`mongoid-config-options-all`

Reviewer: s: 'view an error' is a little vague as to what actually happens

```suggestion
  the server. If a duplicate index is declared, the server raises an error.
  To learn more, see the :ref:`mongoid-config-options-all`
```
